sycl/test-e2e/Matrix/Inputs/joint_matrix_bfloat16_impl.hpp

```diff
@@ -6,6 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <sycl/stream.hpp>
+
```

Comment:
Please, do code cleanup from unnecessary changes.
There are multiple debug leftovers in the file or changes which removed existing functionality.
Please, carefully compare the diff and update the code, so the diff contains only changes that are necessary for the fix.
Thanks.